ddocs/medic/_attachments/translations/messages-en.properties

```diff
@@ -1242,4 +1242,5 @@ contact.profile.imam_history = IMAM History
 contact.profile.last_treatment = Last treatment program
 contact.profile.exit_date = Exit date
 contact.profile.weight_at_exit = Weight at exit (Kg)
-contact.profile.height_at_exit = Height at exit (cm)
\ No newline at end of file
+contact.profile.height_at_exit = Height at exit (cm)
+configuration.user.replication.limit.exceeded = This user would replicate {{total_docs}} docs, which exceeds the recommended limit. Do you wish to proceed?
```

Comment:
As above, it'd be useful to have the recommended limit included here, eg: `... exceeds the recommended limit of {{limit}}...`

Reply:
done.